docs/src/examples/Marquee.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { useState } from 'react';
+import {
+  Image,
+  Pressable,
+  StyleSheet,
+  Text,
+  View,
+  ViewStyle,
+} from 'react-native';
+import 'react-native-gesture-handler';
+import Animated, {
+  SharedValue,
+  useAnimatedStyle,
+  useFrameCallback,
+  useSharedValue,
+} from 'react-native-reanimated';
+
+type MeasureProps = React.PropsWithChildren<{
+  onLayout: (width: number) => void;
+}>;
+
+const MeasureElement = ({ onLayout, children }: MeasureProps) => (
+  <Animated.ScrollView
+    horizontal
+    style={marqueeStyles.hidden}
+    pointerEvents="box-none">
+    <View onLayout={(ev) => onLayout(ev.nativeEvent.layout.width)}>
+      {children}
+    </View>
+  </Animated.ScrollView>
+);
+
+type TranslatedElementProps = React.PropsWithChildren<{
+  index: number;
+  offset: SharedValue<number>;
+  childrenWidth: number;
+}>;
+
+const TranslatedElement = ({
+  index,
+  children,
+  offset,
+  childrenWidth,
+}: TranslatedElementProps) => {
+  const style = useAnimatedStyle(() => {
+    return {
+      position: 'absolute',
```

Comment:
our best practice is to extract every static style to stylesheets whenever possible